src/Components/Cards/ProductCard.js

```diff
@@ -0,0 +1,100 @@
+import "./ProductCard.css";
+import { Link } from "react-router-dom";
+import { StarRatings } from "../index";
+
+import FavoriteSharpIcon from "@material-ui/icons/FavoriteSharp";
+
+export default function ProductCard({ item, reducerState, dispatch }) {
+  const checkIfFav = () => {
```

Comment:
This function can be written in one line using ES6 syntax

```suggestion
const checkIfFav = () => reducerState.favourites.find( itemsInFav => itemsInFav.id === item._id ) ? true : false
```

Reply:
converted the function to follow ES6 syntax